app/src/lib/lookups.js

```diff
@@ -1,5 +1,20 @@
 import {TOKEN_TYPE, flatten_sentence} from './parser/token'
 
+// TODO move whole token pipeline to the server
```

Comment:
I'd like to move the whole token pipeline serverside but I figured this PR has enough for now. Doing this will eliminate the extra api lookup.